packages/frontend/helpers/profile-mapper.ts

```diff
@@ -116,11 +120,21 @@ export const updateDtoMapper = ({
     }
   });
 
+  const updatedPhone = ({ phone }: { phone: string }): string | null => {
+    if (!phone) return null;
+    const isPhone = PHONE.test(phone);
+    if (isPhone) return phone;
+    const isPersonalPhone = PERSONAL_PHONE.test(phone);
+    if (isPersonalPhone) {
+      return DefaultInpValue.PHONE + phone;
+    }
```

Comment:
will it broke if phone is not PHONE and not PersonalPhone?

Reply:
Ivan, i added replace to Joi schema, so it has only one format. 660231547, for example. All other checks - for protection, just in case

Reply:
But on server I added PHONE pattern for validation

Reply:
Ok, got it